src/random.h

```diff
@@ -89,6 +89,14 @@ void GetStrongRandBytes(unsigned char* buf, int num) noexcept;
  */
 void RandAddPeriodic();
 
+/**
+ * Gathers entropy from the low bits of the time at which events occur. Should
+ * be called with a few bytes describing the event at the time an event occurs.
```

Comment:
This comment seems outdated now.

Reply:
Heh, I figured I'd leave it, since it doesn't actually matter what the 4 bytes are, they just have to be aligned and typed as a u32.